renderer.py

```diff
@@ -1668,7 +1668,7 @@ def makeChunkVertices(self, ch):
 
             flatcolors = level.materials.flatColors[blockTypes, ch.Data[blockIndices] & 0xf][:, numpy.newaxis, :]
             for mod_mats in mod_materials:
```

Comment:
Shall be `for mod_mats in mod_materials.values()`, like in other places...
(My bad, I'll change this.)